From 00c85c88b846ca282df303dedcf5d61b52d7ba2d Mon Sep 17 00:00:00 2001
From: Bhaskara Budiredla <bbudiredla@marvell.com>
Date: Fri, 17 Jul 2020 19:16:51 +0530
Subject: [PATCH 601/767] drivers: gicv3: removes the LPI tables warning print

commit db2e7b5d35b0a5ead3819486fda6345298a3a8fc from
git@git.assembla.com:cavium/WindRiver.linux.git

This commit removes the warning message which can
provide incorrect impression on some silicons.

Change-Id: Iecd9318ed1a1b899135740679df4055149bf9a1d
Signed-off-by: Bhaskara Budiredla <bbudiredla@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/32333
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/irqchip/irq-gic-v3-fixes.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/irqchip/irq-gic-v3-fixes.c b/drivers/irqchip/irq-gic-v3-fixes.c
index ee89ac63c70d..e9f7548ba7fe 100644
--- a/drivers/irqchip/irq-gic-v3-fixes.c
+++ b/drivers/irqchip/irq-gic-v3-fixes.c
@@ -181,7 +181,6 @@ struct page *its_prop_alloc_pages(struct rdists *gic_rdists,
 		gic_rdists->prop_table_va = vaddr;
 		prop_page = NULL;
 	} else {
-		pr_warn("Failed to allocate LPI tables in atf\n");
 		prop_page = alloc_pages(gfp_flags, get_order(prop_tbl_sz));
 		if (!prop_page)
 			return NULL;
-- 
2.31.1

